src/askui_runner/modules/core/infrastructure/runner/askui.py

```diff
@@ -103,7 +100,10 @@ def setup(self, dir_path: str) -> None:
     def run_workflows(self) -> RunWorkflowsResult:
         # TODO Determine how this was ended based on exit code, configure jest accordingly, things in file system etc. and with what exit code runner should end --> Use that exit code to set the status
         # TODO Differentiate of failure of runner and failure of workflows and workflows not available and worklows erroneous (including not parseable)
-        wait_for_controller_to_start()
+        wait_for_controller_to_start(
+            host=self.config.controller.host,
```

Comment:
Somehow passing the host like this leads to `host` being empty when used in `is_port_open()`

This config will work: 
```yml
...
runner:
  ...
  controller:
    host: "127.0.0.1"
    port: 7000
```

This will not work

```yml
...
runner:
  ...
  controller:
    host: 127.0.0.1
    port: 7000
```

Reply:
We fix this in another PR